platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -18,13 +18,15 @@
 except ImportError as e:
     raise ImportError(str(e) + "- required module not found")
 
-#components definitions
-COMPONENT_BIOS = "BIOS"
-COMPONENT_CPLD = "CPLD"
+EMPTY = ''
+ZERO = '0'
+COMMA = ','
```

Comment:
We don't need `COMMA`. 
It is not referenced. Even it is, to use the literal constant `,` is more straightforward IMO.

Reply:
@stephenxs fixed